data/txHandler.go

```diff
@@ -263,12 +263,6 @@ func (handler *TxHandler) processIncomingTxn(rawmsg network.IncomingMessage) net
 // Note that this also checks the consistency of the transaction's group hash,
 // which is required for safe transaction signature caching behavior.
 func (handler *TxHandler) checkAlreadyCommitted(tx *txBacklogMsg) (processingDone bool) {
-	txids := make([]transactions.Txid, len(tx.unverifiedTxGroup))
-	for i := range tx.unverifiedTxGroup {
-		txids[i] = tx.unverifiedTxGroup[i].ID()
-	}
-	logging.Base().Debugf("got a tx group with IDs %v", txids)
-
```

Comment:
this code section was just printing transaction ids.. since it's not needed, I removed it.